js/src/components/paid-ads/ads-campaign.js

```diff
@@ -91,13 +115,23 @@ export default function AdsCampaign( {
 				countrySelectHelperText={ helperText }
 				formProps={ formContext }
 			/>
-			<BudgetSection
-				formProps={ formContext }
-				disabled={ disabledBudgetSection }
-				countryCodes={ formContext.values.countryCodes }
-			>
-				<CampaignPreviewCard />
-			</BudgetSection>
+			{ ! loading && (
```

Comment:
@ankitrox This behaviour is different from what we currently have, i.e having a spinner and loading state while the data is being loaded. Is there a particular reason why we need to wait for the recommendations?